lib/jekyll-readme-index/generator.rb

```diff
@@ -26,15 +28,38 @@ def readme
     end
 
     def index?
-      (site.pages + site.static_files).any? { |file| file.url =~ INDEX_REGEX }
+      relative_index?(readme)
     end
 
     def page
+      relative_page(readme)
+    end
+
+    def readmes
+      site.static_files.each_with_object([]) do |file, memo|
+        memo << file if file.relative_path =~ READMES_REGEX
+      end
+    end
+
+    def relative_index?(readme)
```

Comment:
hmm, maybe. actually I think this would all work cleaner with a static file decorator, but it seemed like overkill for the application